### DISCUSSION TITLE
FPS display displays higher "Draw" than "Update" fps

### DISCUSSION THREAD

**Discussion by Morilli** [ppy/osu#19329] [Q&A] [answered]
The newly added FPS display has weird behavior when "Multithreaded" is enabled and `Update` fps drop below ~10 (my monitor is 60hz, not sure if that matters): The displayed draw fps (which are also the fps displayed when not hovering the corner display) appear much higher than they actually are:
![image](https://user-images.githubusercontent.com/35152647/180602921-d437807a-07af-40f5-b6a0-a5b4a0eb94e3.png)
In this scenario, "real" fps in terms of visual screen updates were definitely 6, not 12.
This doesn't happen in single threaded mode fwiw (6 would get displayed for both update and draw fps).

**Comment by peppy** [comment, marked as answer]
Dunno, it's an average and your numbers are probably too low to be worth even bothering about. Please try with a higher frame rate.

**Reply by Morilli** [reply]
Higher framerates display just fine; this is just an issue specifically once update fps drop below 10 fps (draw fps immediately align again once update fps return to 11+).
I can see how this is not a priority to look into though, because such low fps are unlikely to be hit in normal scenarios.
